content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -8,6 +8,9 @@ content_template: templates/concept
 weight: 100
 ---
 {{% capture overview %}}
+
+{{< note >}} **Note:** This document is for Kubernetes 1.10. {{< /note >}}
```

Comment:
A better way to indicate this is with a feature state shortcode, which lets us indicate both version and whether the feature is alpha | beta | GA. See https://github.com/kubernetes/website/blob/master/content/en/docs/home/contribute/includes/index.md